unleash-yggdrasil/src/strategy_parsing.rs

```diff
@@ -353,14 +357,30 @@ fn list_constraint(inverted: bool, mut node: Pairs<Rule>) -> RuleFragment {
 
                 match comparator {
                     ContentComparator::In => match context_value {
-                        Some(context_value) => values.contains(&context_value).invert(inverted),
+                        Some(context_value) => values.contains(&context_value),
                         None => false,
                     },
                     ContentComparator::NotIn => match context_value {
-                        Some(context_value) => !values.contains(&context_value).invert(inverted),
+                        Some(context_value) => !values.contains(&context_value),
+                        None => true,
+                    },
+                    ContentComparator::InIgnoreCase => match context_value {
+                        Some(context_value) => {
+                            let needle = context_value.to_lowercase();
+                            values.iter().find(|x| x.to_lowercase() == needle).is_some()
```

Comment:
Thank you Clippy! This can be written as

```suggestion
       values.iter().any(|x| x.to_lowercase() == needle)
```

Which avoids the `is_some()` check